client/src/containers/PublishDetails/index.js

```diff
@@ -1,16 +1,29 @@
 import { connect } from 'react-redux';
-import { clearFile, startPublish } from '../../actions/publish';
+import { clearFile, startPublish, abandonClaim } from '../../actions/publish';
 import View from './view';
 
-const mapStateToProps = ({ channel, publish }) => {
+const mapStateToProps = ({ show, publish }) => {
+  // select request info
+  const requestId = show.request.id;
```

Comment:
You've repeated this code a few times for getting the `asset`, I'd suggest creating a util method